stylesheets/_component.tables.scss

```diff
@@ -121,6 +121,7 @@
 .table-container {
     margin-bottom: 24px;
     overflow-x: auto;
+    position: relative;
```

Comment:
Does this need adding here, if it's just needed for the horizontal changes could it live under `.table--horizontal` so as not to affect other tables?